gpc-api-facade/src/integrationTest/java/uk/nhs/adaptors/pss/gpc/controller/AcknowledgeRecordControllerIT.java

```diff
@@ -138,9 +152,20 @@ public void sendAcknowledgeRequestWithMissingConfirmationResponseHeader() throws
                 .andExpect(content().json(expectedResponseBody));
     }
 
+    // Helper Methods
```

Comment:
I think we don't need this comments as they a bit clutter it here, the method is simple and it is very easy to see that it is a preparing/helper method for the test.